File: practice-challenges/add-all-nums/addNums.js
/* Solution for Add All Numbers. */

function addNumbers(nums) {
  let total = 0;
  for (let num of nums) {
    total += num;
  }

  return total;
}

// Tests

// 1 + 1 + 1 = 3
console.log(addNumbers([1, 1, 1]));

// No numbers to add, return 0
console.log(addNumbers([]));

File: practice-challenges/add-all-nums/add_nums.py
"""Solution for Add All Numbers."""


def add_numbers(nums):
    total = 0
    for num in nums:
        total += num

    return total


# Tests

# 1 + 1 + 1 = 3
print(add_numbers([1, 1, 1]))

# No numbers to add, return 0
print(add_numbers([]))